src/instrumentation_measurement/measurements.py

```diff
@@ -82,21 +82,23 @@ def measure(self, sampleSize, mode="all", evm="geth", nSamples=1):
     openethereum = "openethereum"
     openethereum_ewasm = "openethereum_ewasm"
     evmone = "evmone"
+    nethermind = "nethermind"
 
     measure_total = "total"
     measure_all = "all"
     trace_opcodes = "trace"
+    benchmark_mode = "benchmark"
 
-    if not self._check_clocksource():
-      print("clocksource should be tsc, found something different. See docker_timer.md somewhere in the docs")
-      return
+    # if not self._check_clocksource():
```

Comment:
why this? this is very useful for my current work

Reply:
Corrected